cli/src/semgrep/config_resolver.py

```diff
@@ -256,11 +257,14 @@ def is_registry_url(self) -> bool:
         return self._origin == ConfigType.REGISTRY
 
     def _project_metadata_for_standalone_scan(self) -> out.ProjectMetadata:
-        repo_name = os.environ.get("SEMGREP_REPO_NAME") or "unknown"
-        if repo_name is None:
+        repo_name = os.environ.get("SEMGREP_REPO_NAME")
+
+        if repo_name is None and is_policy_id(self._config_path):
```

Comment:
What's the motivation to make this optional here? 
Do we need to ensure that it's required for `--config code`?
Why not just make it required for all so that we have the data available in the backend later if/when we want it?

Reply:
see the comments in the parent PR